rts/rts.cabal

```diff
@@ -613,11 +613,13 @@ library
           Jumps_V32.cmm
           Jumps_V64.cmm
 
+      -- we always link against libffi, even without libffi-adjustors
+      extra-libraries: ffi
+      extra-libraries-static: ffi
```

Comment:
Should we leave a comment here (in the rts) or somewhere else explaining what GHC needs ffi for?

Reply:
Sure, done.